app/assets/javascripts/actions/story.js

```diff
@@ -108,13 +108,14 @@ export const confirmBeforeSaveIfNeeded = async (story, confirm, needConfirmation
 }
 
 export const showHistory = (storyId, from) =>
-  async (dispatch, getState, { Story }) => {
-    const { stories } = getState();
+  async (dispatch, getState, { Story, History }) => {
+    const { stories, users } = getState();
     const story = Story.findById(Story.withScope(stories, from), storyId)
     dispatch(loadHistory(story.title))
     try {
       const activities = await Story.getHistory(story.id, story.projectId)
-      dispatch(receiveHistory(activities, story.title))
+      const unserializedActivities = History.deserializeHistory(activities, users)
```

Comment:
What do you think about moving this call one step inner and put it inside `Story.getHistory`? It's ok to pass `users` to this method.

Reply:
For sure.